<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphtool</class>
 <widget class="QMainWindow" name="graphtool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>graphtool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNavigation"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Vertex"/>
   <addaction name="actionDelete_Vertex"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_arc"/>
   <addaction name="actionDelete_arc"/>
   <addaction name="separator"/>
   <addaction name="actionBFS"/>
   <addaction name="actionDfS"/>
   <addaction name="separator"/>
   <addaction name="actionDijkstra"/>
   <addaction name="actionBellman_ford"/>
   <addaction name="separator"/>
   <addaction name="actionMatrice_d_adjacence"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Vertex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add vertex</string>
   </property>
   <property name="toolTip">
    <string>Click on the canvass to add a vertex</string>
   </property>
  </action>
  <action name="actionDelete_Vertex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delete vertex</string>
   </property>
  </action>
  <action name="actionAdd_arc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add arc</string>
   </property>
   <property name="toolTip">
    <string>Select two vertexes to create an arc between them</string>
   </property>
  </action>
  <action name="actionDelete_arc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delete arc</string>
   </property>
   <property name="toolTip">
    <string>Select an arc to delete it</string>
   </property>
  </action>
  <action name="actionBFS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BFS</string>
   </property>
   <property name="toolTip">
    <string>Broad First search</string>
   </property>
  </action>
  <action name="actionDfS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DFS</string>
   </property>
   <property name="toolTip">
    <string>Depth First Search</string>
   </property>
  </action>
  <action name="actionDijkstra">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dijkstra</string>
   </property>
   <property name="toolTip">
    <string>Shortest path (Dijkstra Algorithm)</string>
   </property>
  </action>
  <action name="actionBellman_ford">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bellman-ford</string>
   </property>
   <property name="toolTip">
    <string>Shortest path(Bellman-ford Algorithm)</string>
   </property>
  </action>
  <action name="actionMatrice_d_adjacence">
   <property name="text">
    <string>Matrice d'adjacence</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionCon_tac">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
  <action name="actionNavigation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Navigation</string>
   </property>
   <property name="toolTip">
    <string>Navigation mode to move and select objects</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
